Hunting Queries/Corelight/CorelightObfuscatedBinary.yaml

```diff
@@ -0,0 +1,26 @@
+id: a518ea0c-f87a-4a79-8f45-2b5b6f44bd7f
+name: Corelight - Obfuscated binary filenames
+description: |
+  'Query searches for downloaded obfuscated binary file names.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL

```

Comment:
If we are using parser in query datatype should be parser name. So could you please update the datatype name to Corelight.

Reply:
updated